lib/directories.zsh

```diff
@@ -24,10 +24,11 @@ alias rd=rmdir
 alias d='dirs -v | head -10'
 
 # List directory contents
-alias lsa='ls -lah'
-alias l='ls -lah'
+# alias lsa='ls -lah'
+alias l='ls'
```

Comment:
Im not convinced that it's a big enough gain to justify breaking user's aliases. It is fairly sane though